roles/installer/molecule/default/verify.yml

```diff
@@ -8,3 +8,12 @@
     installer_no_colors: false
   roles:
     - installer
```

Comment:
should we pass the same options as in `converge.yml` here? currently, we don't (not that the installer cares much…)